ios/.gitignore

```diff
@@ -37,7 +37,4 @@ fastlane/README.md
 
 # IPA and Dsyms
 Runner.app.dSYM.zip
-Runner.ipa
-
-# Google Services Info.plist
-Runner/GoogleService-Info.plist
\ No newline at end of file
+Runner.ipa
```

Comment:
Why do we want to remove this from gitignore?

Reply:
It was outdated because we were previously using Fastlane gym, which creates an .ipa file in the root folder. Now that we are using `flutter build ipa`, we don't need it in .gitignore anymore